src/vendorSpecific.ts

```diff
@@ -0,0 +1,30 @@
+import { postgresSequenceToSerialType } from './utils';
+
+const modifiers: modifiersType = {
+    import: {
```

Comment:
If the issue is that it is happening on Import, then I'd suggest executing the modifier on import instead of on export.